### PR TITLE
chore: remove duplicate code in tests of leverage module

### PR DESCRIPTION
**Pull request by gsk967**
## Description

closes: #1232 

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Review by toteki — APPROVED** [review]

**Comment by toteki** [comment]
There are some tests that need updating with the altered token params (the first one is `QueryRegisteredTokens` in `x/leverage/client` tests) but there will be others in the same location once that error is cleared.

Run `make test-unit` locally to catch

**Comment by gsk967** [comment]
@toteki  Please can you check the calculations part in test cases ?

File: tests/e2e/e2e_setup_test.go [hunk 1]
```diff
@@ -250,26 +251,9 @@ func (s *IntegrationTestSuite) initGenesis() {
...
-		EnableMsgSupply:        true,
-		EnableMsgBorrow:        true,
-		Blacklist:              false,
-		MaxCollateralShare:     sdk.MustNewDecFromStr("1"),
-		MaxSupplyUtilization:   sdk.MustNewDecFromStr("1"),
-		MinCollateralLiquidity: sdk.MustNewDecFromStr("0"),
-		MaxSupply:              sdk.NewInt(100000000000),
-	})
+	leverageGenState.Registry = append(leverageGenState.Registry,
+		fixtures.Token(appparams.BondDenom, appparams.DisplayDenom, 6),
```
Comment (robert-zaremba) [inline tests/e2e/e2e_setup_test.go:255]: if needed to customize more, we can add one more function.

**Review by robert-zaremba — APPROVED** [review]
lgtm, check with @toteki for final approval.

**Review by toteki — APPROVED** [review]
Checked calculations and updated some of the comments where amounts were changed.

Also reverted and left comment in `suite.go` about why we were stopping test of first error. 

Basically, if using `s.Run()`, when the error is in a transaction (e.g. `MsgSupplyCollateral`) almost all steps afterwards which depend on it also print errors. This made scrolling through the false errors to find the real one difficult, so I make `testQuery.Run() and testTransaction.Run()` to fix it some time ago.

**Pull request merged**